ding_blog.info

```diff
@@ -13,6 +13,7 @@ dependencies[] = secure_permissions
 dependencies[] = taxonomy
 dependencies[] = text
 dependencies[] = ting_reference
+stylesheets[all][] = css/ding_blog.css
```

Comment:
in this case the css file will be loaded on all pages. But we have to do it only on separate ones: `/users/%staff_user`, `blog`, etc.